src/template_finder.py

```diff
@@ -190,14 +190,20 @@ def search(
             template_match = TemplateMatch()
             scale = scales[count]
 
+            mask = None
```

Comment:
Can you add one short comment above this code what it does. something like `# Use alpha channel to create a mask for alpha == 0. Since it does something different than the name suggests in the first place. So whenever code is not "self documented" e.g. by variable names, it makes actually sense to add comments.